src/parameterizations/lateral/MOM_thickness_diffuse.F90

```diff
@@ -1614,10 +1614,16 @@ subroutine thickness_diffuse_full(h, e, Kh_u, Kh_v, tv, uhD, vhD, cg1, dt, G, GV
       MEKE%GM_src(i,j) = MEKE%GM_src(i,j) + Work_h
     endif ; endif
     if (skeb_use_gm) then
-      h_tot = sum(h(i,j,1:nz))
+      h_tot = 0.0
+      do k=1,nz
+        h_tot = h_tot + h(i,j,nz)
+      enddo
```

Comment:
`h_tot` doesn't appear to be used anywhere. Is this computation necessary?

Reply:
Agreed - I think we can just remove `h_tot` since it doesn't appear to be used anywhere.

Reply:
My latest commit removes this.